action.yml

```diff
@@ -28,10 +28,7 @@ branding:
 inputs:
   version:
     description: |-
-      Version of buf to download and add to the PATH.
-      Example:
-        with:
-          version: 1.32.1
+      Version of buf to use. It will be download and added to the PATH if this version doesn't already exist.
```

Comment:
It will fail if the local version is different to the version specified. Although I think this is an edge case. I think the version example is good to show its the semver version without the 'v' sufffix (although that might work as I do some version coercing). I think we should also mention its an explicit verison.